api/tests/unit/domain/services/organization-service_test.js

```diff
@@ -50,67 +51,67 @@ describe('Unit | Service | OrganizationService', () => {
     let organizationId;
     let targetProfilesOwnedByOrganization;
     let targetProfileSharesWithOrganization;
+    let targetProfileOrganizationCanUse;
     let publicTargetProfiles;
 
     beforeEach(() => {
       organizationId = 1;
       targetProfilesOwnedByOrganization = [domainBuilder.buildTargetProfile({ organizationId, isPublic: false })];
       targetProfileSharesWithOrganization = domainBuilder.buildTargetProfile({ isPublic: false });
       publicTargetProfiles = [domainBuilder.buildTargetProfile({ isPublic: true })];
+
       const targetProfileShares = [{
         targetProfile: targetProfileSharesWithOrganization
       }];
+
       const organization = domainBuilder.buildOrganization({ id: organizationId, targetProfileShares });
+      targetProfileOrganizationCanUse = concat(publicTargetProfiles, targetProfilesOwnedByOrganization);
 
-      sinon.stub(targetProfileRepository, 'findPublicTargetProfiles').resolves(publicTargetProfiles);
-      sinon.stub(targetProfileRepository, 'findTargetProfilesOwnedByOrganizationId').resolves(targetProfilesOwnedByOrganization);
+      sinon.stub(targetProfileRepository, 'findAllTargetProfileOrganizationCanUse').resolves(targetProfileOrganizationCanUse);
       sinon.stub(organizationRepository, 'get').resolves(organization);
     });
 
-    it('should return an array of target profiles', () => {
+    it('should return an array of target profiles', async () => {
```

Comment:
ma proposition comme on utilise la method ` instanceof`
- should return an array of type target profile

Reply:
Done.